systemvm/patches/debian/config/opt/cloud/bin/configure.py

```diff
@@ -479,13 +479,13 @@ def process(self):
 
     def deletevpn(self, ip):
         logging.info("Removing VPN configuration for %s", ip)
-        CsHelper.execute("ipsec auto --down vpn-%s" % ip)
-        CsHelper.execute("ipsec auto --delete vpn-%s" % ip)
+        CsHelper.execute("ipsec down vpn-%s" % ip)
+        CsHelper.execute("ipsec down vpn-%s" % ip)
```

Comment:
Thanks for the explanation @jayapalu 

I have tested this manually on a 4.5.2 installation by backporting the scripts directly inside a VR and can say that it fundamentally works. Not sure if I have time to spin up a new setup before freeze though.

All in all it LGTM, but I would want someone with a more extensive test than myself to vouch as well